Sources/AblyAssetTrackingPublisher/State/DefaultSkippedLocationsState.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  Created by Łukasz Szyszkowski on 30/08/2021.
+//
+
+import Foundation
+
+class DefaultSkippedLocatoinsState: PublisherSkippedLocationsState {
```

Comment:
Typo - there is `Locatoins` but should be `Locations` :wink:

Reply:
d174b97ea43d7666a97de45d02ca18a38f0aadaa